content/php-client/http-client.md

```diff
@@ -0,0 +1,48 @@
+# HTTP client abstraction
+
+We use [HTTPPlug](http://httplug.io/) as the HTTP client abstraction layer. 
+Thanks to it, you are free to use the HTTP client implementation and PSR7 implementation that you fulfil your needs.
+ 
+For example, if you already are using Guzzle v5 in an existing project, you can use it with Akeneo PHP client as well. 
```

Comment:
both are correct, I've checked it. 
An english speaker will tell us what is the best :)